src/steps/Microsoft.DotNet.UpgradeAssistant.Steps.Razor/HelperReplacement.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Text;
+
+namespace Microsoft.DotNet.UpgradeAssistant.Steps.Razor
+{
+    public class HelperReplacement
```

Comment:
This type seems like it could be a general purpose text replacement holder

Reply:
Yea, I considered that. I already have a `TextReplacement` type though for source replacement which functions a little differently from this one (it contains information on text in a generated source file that needs to be replaced on a particular line in a Razor document).

Maybe I'll rename that one to something else, though (MappedTextReplacement?) so that this can be TextReplacement.

Reply:
Ok, I've updated the file names to be (hopefully) more sensible.